ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -36,13 +36,19 @@
     - PyYAML  # ipa-vagrant-ci dependency
     - vim
     - NetworkManager
-    - python3-paramiko
 
 - name: install pip dependencies
   pip:
+    executable: pip3
```

Comment:
This breaks backwards compatibility. Before, it was possible to run ipa tests with python2 -- if you don't install this package with pip as well, it won't be possible to run tests with python 2 anymore.

I'd either install this with both pip and pip3, or bump the template version to 0.2.x to signify some compatibility/API has been changed.

Reply:
I see we are not on the same page here.
I thought we would have new templates only for python 3 stuff. I mean, we could stay with 0.1.4 to run tests with py2, and the new templates would be only to help PR 1112. Once all tests were migrated to py3, we would drop the support for 0.1.4.

However, if this is not the idea, I'll add the installation with `pip` as well.

Reply:
The disk space on the runners is very limited (less than 20GB for templates, each template takes ~2.8GB). Keep in mind each branch can (and may have to) have a separate template. Also, we may want to test on multiple distros in the future. There should never be more than a single supported version for branch/distro, we simply don't have the space for it.

Reply:
Also, any breaking changes or significant updates should bump either x or y in x.y.z version format. The default z bump is intended primarily for minor changes and package updates. Supporting py2 in 0.1.4 and not supporting it in 0.1.6 is not a good idea.